examples/typescript/client/index.ts

```diff
@@ -2,12 +2,16 @@ import "babel-polyfill";
 import { Server } from "../endpoints";
 import { server as serverUntyped } from "@wildcard-api/client";
 
-export const server: Server = serverUntyped;
+export const server = serverUntyped as Server;
```

Comment:
@brillout It would be nice if we could do `const server: Server = serverUntyped;` instead of casting. Any idea how to improve this?

Reply:
Agree. We will need to cast at some point or another though. But I'm thinking maybe we could hide the casting from the user.

Reply:
I suggest to spend more time on this when more people are using it.

Reply:
Agree.